networkmanager_dmenu

```diff
@@ -481,14 +492,13 @@ def get_selection(eths, aps, vpns, wgs, gsms, blues, wwan, others, saved):
         sys.exit()
 
     if rofi_highlight is False:
-        action = [i for i in eths + aps + vpns + wgs + gsms + blues + wwan + others + saved
+        action = [i for i in all_actions
```

Comment:
This now includes the empty `Action('', None)` which it didn't before - but I think it shouldn't make a difference, as `sel` cannot be an empty string (would've exited above), so the condition below can never be true for empty actions anyways.